xml/System.Net.Sockets/LingerOption.xml

```diff
@@ -230,9 +230,9 @@
         <Parameter Name="comparand" Type="System.Object" Index="0" FrameworkAlternate="net-8.0" />
       </Parameters>
       <Docs>
-        <param name="comparand">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="comparand">The <see cref="T:System.Object"/> to compare with the current <see cref="T:System.Net.Sockets.LingerOption"/>.</param>
+        <summary>Determines whether the specified <see cref="T:System.Object"/> is equal to the current <see cref="T:System.Net.Sockets.LingerOption"/>.</summary>
+        <returns><see langword="true"/> if the specified <see cref="T:System.Object"/> is equal to the current <see cref="T:System.Net.Sockets.LingerOption"/>; otherwise, <see langword="false"/>.</returns>
```

Comment:
```suggestion
        <returns><see langword="true"/> if the specified object is equal to the current <see cref="T:System.Net.Sockets.LingerOption"/> instance; otherwise, <see langword="false"/>.</returns>
```